R/AllS4.R

```diff
@@ -3,21 +3,21 @@ if ("package:data.table" %in% search()) stopf("data.table package loaded. When d
 
 ## Allows data.table to be defined as an object of an S4 class,
 ## or even have data.table be a super class of an S4 class.
-setOldClass(c('data.frame'))
-setOldClass(c('data.table', 'data.frame'))
+methods::setOldClass(c('data.frame'))
```

Comment:
I would just import them once at start of the script by

setOldClass = methods::setOldClass

Reply:
that has the effect of basically making this `importFrom(methods, setOldClass)`, whereas `setOldClass()` is only a build-time dependency and not needed by the installed package at all.